experiments/experiments.go

```diff
@@ -357,7 +357,7 @@ type Variant struct {
 type UnknownExperimentError string
 
 func (name UnknownExperimentError) Error() string {
-	return fmt.Sprintf("experiments: experiment with name %s unknown", name)
+	return "experiments: experiment with name" + string(name) + " unknown"
```

Comment:
You want a space after “name”

Reply:
> `%v` would cause infinite recursion but `%s` _should_ be fine?

I would have thought so too but I was running into this bug on missing experiments.